sources/editor/Stride.Assets.Presentation/StrideDefaultAssetsPlugin.cs

```diff
@@ -93,7 +93,7 @@ public static void LoadDefaultTemplates()
         {
             // Load templates
             // Currently hardcoded, this will need to change with plugin system
-            foreach (var packageInfo in new[] { new { Name = "Stride.Assets.Presentation", Version = StrideVersion.NuGetVersion }, new { Name = "Stride.SpriteStudio.Offline", Version = StrideVersion.NuGetVersion }, new { Name = "Stride.Samples.Templates", Version = Stride.Samples.Templates.ThisPackageVersion.Current } })
+            foreach (var packageInfo in new[] { new { Name = "stride.assets.presentation", Version = StrideVersion.NuGetVersion }, new { Name = "stride.spritestudio.offline", Version = StrideVersion.NuGetVersion }, new { Name = "stride.samples.templates", Version = Stride.Samples.Templates.ThisPackageVersion.Current } })
```

Comment:
If this one didn't entirely solve your issues, best to open a new pr just for this so we can have a discussion around the issue you're having. I also have issues building the tests you mentioned by my pc has been crashing a lot lately so can't really give you anything definite around this one.

Reply:
Made the PR and reverted this change also. Bit off topic but as I revert changes, should I mark these conversations as resolved?

Reply:
Yep, but don't worry too much about that feature, it doesn't matter too much :)